tests/fixtures/data/invoices.json

```diff
@@ -144,19 +144,19 @@
             "data": {
                 "price": 0.2,
                 "products": [
-                    2
+                    1
```

Comment:
By the way: does it fix the issues we had, relying on object id? Probably `ViewTestMixin` needs a bigger refactor? As for now, it still relies on object id.

Reply:
It is still relying on object id and it should rely on due to some tests required to retrieve obj by id (now integer id, future unique id)